frontend/src/views/web/partials/dyn-header.html

```diff
@@ -7,7 +7,7 @@
 
       <ul class="right hide-on-med-and-down">
         <li><a class="waves-effect waves-dark" ui-sref="home" ui-sref-active="active">Home</a></li>
-        <li><a class="waves-effect waves-dark" ui-sref="web.dashboard" ui-sref-active="active">Dashboard</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark" ui-sref="web.dashboard" ui-sref-active="active">Dashboard</a></li>
```

Comment:
@aka-jain : I think this list item should be after the comment (#L13). That make sense with code readability.